recipes/heavydb/meta.yaml

```diff
@@ -0,0 +1,575 @@
+        - test -f ${PREFIX}/{{ install_base }}/bin/calcite-1.0-SNAPSHOT-jar-with-dependencies.jar  # [unix]
+        - if not exist %PREFIX%\bin\heavyai-utility-{{ base_version }}.jar exit 1  # [win]
+        - if not exist %PREFIX%\bin\heavyai-jdbc-{{ base_version }}.jar exit 1  # [win]
+        - if not exist %PREFIX%\bin\calcite-1.0-SNAPSHOT-jar-with-dependencies.jar exit 1  # [win]
+        # Unspecified
+        # startheavy and insert_sample_data are bash scripts
+        - test -f ${PREFIX}/{{ install_base }}/bin/startheavy  # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/bin/heavydb_insert_sample_data  # [unix]
+        #
+        - test -f ${PREFIX}/{{ install_base }}/bin/generate_cert  # [unix]
+        - if not exist %PREFIX%\bin\generate_cert exit 1  # [win]
+
+  - name: heavydb
+    version: {{ version }}
+    script: build-heavydb.sh   # [linux64]
+    script: build-heavydb.bat  # [win]
+    build:
+      string: {{ build_string_ext }}
+      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0")]
+      skip: true  # [osx]
+      skip: true  # [win and cuda_compiler_version != "None"]
+      missing_dso_whitelist:
+        - '*/libcuda.*'  # [cuda_compiler_version not in (undefined, "None")]
+      track_features:
+        {{ "- arrow-cuda" if cuda_enabled else "" }}
+      ignore_run_exports:
+        - cudatoolkit    # [cuda_compiler_version not in (undefined, "None")]
+        - fmt
+        - gflags
+        - glog
+        - libclang-cpp
+        - libkml
+        - ncurses
+        - openldap
+        - zlib
+      run_exports:
+        - {{ pin_subpackage('heavydb',  max_pin='x.x.x') }}
+    requirements:
+      build:
+        # c compiler is specified here to get run constraint pins correct, presumably..
+        - {{ compiler('c') }}
+        - {{ compiler('cgo') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler("cuda") }}  # [cuda_compiler_version not in (undefined, "None")]
+        - clangdev {{ llvm_version }}
+        # clang++ is used for generating the bytecodes of extension functions
+        - clangxx {{ llvm_version }}
+        - cmake
+        - llvmdev {{ llvm_version }}
+        - make
+        - ninja
+        - maven
+      host:
+        - arrow-cpp ={{ arrow_version }}=*{{ build_ext }}
+        - bisonpp  # [unix]
+        - blosc
+        - boost-cpp
+        - clangdev {{ llvm_version }}
+        - double-conversion
+        - flex  # [unix]
+        - fmt
+        - gflags
+        - glog
+        - llvmdev {{ llvm_version }}
+        - llvm {{ llvm_version }}
+        - libarchive
+        - libclang-cpp {{ llvm_version }}
+        - libevent  # [win]
+        - libgdal
+        - libkml
+        - libpng
+        - librdkafka
+        - libzlib
+        - ncurses   # [unix]
+        - openldap  # [unix]
+        # Workaround https://github.com/mamba-org/boa/issues/119 :
+        - openssl 1.1.1*
```

Comment:
This might conflict with future iterations of the openssl migration. I wouldn't pin it here. boa's issue should have been fixed by now, too.